web/ts/global.ts

```diff
@@ -105,6 +105,64 @@ function initHiddenCourses() {
     }
 }
 
+// Adapted from https://codepen.io/harsh/pen/KKdEVPV
+function timer(expiry: string, regular: boolean, leadingZero: boolean) {
+    return {
+        expiry: new Date(expiry),
+        remaining: null,
+        init() {
+            const offsetInMinutes = regular ? 10 : 0;
```

Comment:
For some reason, the timer on the start page has an offset of 20 minutes instead of 10, any idea why? :thinking:

Reply:
Not really, but moving the time calculation outside of the `init` function somehow solves it.